ACPs/103-dynamic-fees/README.md

```diff
@@ -0,0 +1,135 @@
+```text
+ACP: 103
+Title: Add Dynamic Fees to the X-Chain and P-Chain
+Author(s): Dhruba Basu (https://github.com/dhrubabasu), Alberto Benegiamo (https://github.com/abi87), Stephen Buttolph (https://github.com/StephenButtolph)
+Discussions-To: <GitHub Discussion URL (POPULATED BY MAINTAINER, DO NOT SET)>
+Status: Proposed
+Track: Standards
+```
+
+## Abstract
+
+Introduce a dynamic fee mechanism to the X-Chain and the P-Chain. Preview a future transition to a multidimensional fee mechanism.
+
+## Motivation
+
+Under a fixed fee mechanism, there is a greater likelihood of higher network congestion. During periods of high demand, there is no economic incentive for users to withhold issuance of their transactions, extending the period of high demand.
```

Comment:
> Under a fixed fee mechanism, there is a greater likelihood of higher network congestion.
I only think of escalating fees in the context of both block supply + demand (not just demand). I think "greater likelihood of network congestion" implies this but doesn't satisfy me (sounds like some game of chance).

> During periods of high demand, there is no economic incentive for users to withhold issuance of their transactions, extending the period of high demand.

This implies that "high demand" is the problem. I think that is the wrong mentality. The problem is that the network does not adjust the fee for inclusion to a market clearing price at capacity (both leaving money on the table for the network and leading users to compete over the protocol stack for inclusion).